meson.build

```diff
@@ -1490,11 +1490,16 @@ if want_tpm2 != 'false' and not skip_deps
         tpm2 = dependency('tss2-esys tss2-rc tss2-mu',
                           required : want_tpm2 == 'true')
         have = tpm2.found()
+        tss2_esys3 = dependency('tss2-esys', version : '>= 3.0.0',
+                                required : false)
+        have_esys3 = tss2_esys3.found()
```

Comment:
hmm, does this work:

```meson
tss2_esys3 = tpm2.version().version_compare('>= 3.0.0')
```

Reply:
Yes, it seems to work (wasn't sure because dependency() references 3 packages but they should always have the same version I believe. Let me update the PR.